aicsimageio/tests/writers/test_ome_tiff_writer.py

```diff
@@ -1,80 +1,111 @@
 
 import numpy as np
 
 from aicsimageio.readers.ome_tiff_reader import OmeTiffReader
 from aicsimageio.writers import OmeTiffWriter
 
 
-class TestOmeTiffWriter(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.dir_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "resources")
-        cls.file = os.path.join(cls.dir_path, "ometif_test_output.ome.tif")
-        cls.image = np.random.rand(1, 40, 3, 128, 256).astype(np.uint16)
-        cls.writer = OmeTiffWriter(cls.file, overwrite_file=True)
-        if not os.path.isfile(cls.file):
-            open(cls.file, "a").close()
+filename = "ometif_test_output.ome.tif"
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.writer.close()
-        os.remove(cls.file)
+image = np.random.rand(1, 40, 3, 128, 256).astype(np.uint16)
 
+
+def test_writerShapeComparison(resources_dir):
     """
     Test to check that OmeTiffWriter saves arrays that are reflexive with OmeTifReader
     """
+    writer = OmeTiffWriter(resources_dir / filename, overwrite_file=True)
+    writer.save(image)
+    writer.close()
+
+    with OmeTiffReader(resources_dir / filename) as test_output_reader:
+        output = test_output_reader.data
 
-    def test_writerShapeComparison(self):
-        self.writer.save(self.image)
+    assert output.shape == image.shape[1:]
 
-        with OmeTiffReader(self.file) as test_output_reader:
-            output = test_output_reader.data
+    # os.remove(resources_dir / filename)
 
-        self.assertEqual(output.shape, self.image.shape[1:])
 
+def test_loadAssertionError(resources_dir):
     """
     Test to check if save() will only accept 3, 4, 5 dimensions for data
     """
+    image_to_save = np.ones((1, 2, 3, 4, 5, 6))
+    with pytest.raises(Exception):
+        writer = OmeTiffWriter(resources_dir / filename, overwrite_file=True)
+        writer.save(image_to_save)
 
-    def test_loadAssertionError(self):
-        image_to_save = np.ones((1, 2, 3, 4, 5, 6))
-        with self.assertRaises(Exception):
-            self.writer.save(image_to_save)
 
+def test_overwriteFile(resources_dir):
     """
     Test to check if save() can overwrite a file
     """
+    with OmeTiffWriter(resources_dir / filename, overwrite_file=True) as writer:
+        writer.save(image)
 
-    def test_overwriteFile(self):
-        with OmeTiffWriter(self.file, overwrite_file=True) as writer:
-            writer.save(self.image)
 
+def test_dontOverwriteFile(resources_dir):
     """
     Test to check if save() will raise error when user does not want to overwrite a file that exists
     """
+    with pytest.raises(Exception):
+        with OmeTiffWriter(resources_dir / filename) as writer:
+            writer.save(image)
 
-    def test_dontOverwriteFile(self):
-        with self.assertRaises(Exception):
-            with OmeTiffWriter(self.file) as writer:
-                writer.save(self.image)
 
+def test_noopOverwriteFile(resources_dir):
```

Comment:
Wow, I honestly didn't know this was the functionality and I am not sure I like it... I would prefer to throw a `FileExistsError`. Open to discussion though.